include/gtl/vector.h

```diff
@@ -236,7 +236,7 @@ class base_vector
     }
 
     template< class InputIt >
-    void insert(iterator pos, InputIt first, InputIt last)
+    void insert(iterator pos, InputIt first, InputIt last, const T& value)

```

Comment:
Ahh, this and the next are wrong, but thats my fault.
I need to clean those up, because this is actualy inserting at pos, copying constructing the values from first and last..